package.json

```diff
@@ -42,7 +42,7 @@
   ],
   "license": "MIT",
   "devDependencies": {
-    "@babel/cli": "7.7.5",
+    "@babel/cli": "7.12.1",
```

Comment:
should not be part of this PR

Reply:
Should we rebase `mdx` to sync with `master`?

Reply:
Alright I rebased this branch